src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,8 +1,21 @@
-export const TodoFilter = () => (
+type Props = {
+  onFilterChange: (value: string) => void,
+  onQueryChange: (value: string) => void,
+  query: string,
+};
+
+export const TodoFilter: React.FC<Props> = ({
+  onFilterChange,
+  onQueryChange,
+  query,
+}) => (
   <form className="field has-addons">
     <p className="control">
       <span className="select">
-        <select data-cy="statusSelect">
+        <select
+          data-cy="statusSelect"
+          onChange={(event) => onFilterChange(event.target.value)}
```

Comment:
Let's accept event in already existing function `handleFilterChange`, so we don't need to create a new arrow function here
```suggestion
          onChange={onFilterChange}
          
```